doc/exts/pylint_features.py

```diff
@@ -2,7 +2,8 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 
-"""Script used to generate the features file before building the actual documentation."""
+"""Script used to generate the features file before building the actual
+documentation."""
```

Comment:
Please let's not do that. It's just not worth it. Not everything needs to be perfect.